docs/admin/cluster-large.md

```diff
@@ -36,7 +36,7 @@ Documentation for other releases can be found at
 
 ## Support
 
-At v1.1, Kubernetes supports clusters up to 250 nodes with 30 pods per node and 1-2 containers per pod.
+At v1.2, Kubernetes supports clusters up to 1000 nodes with 30 pods per node and 1-2 containers per pod.
```

Comment:
> We support 30.000 pods per cluster.

Is this really true? I am not sure a 30k-node cluster with one pod per node would work (within our SLO). Are you suggesting I rephrase what is written here? If so, what would you suggest?